hearpreprocess/spoken_digit.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+"""
+Pre-processing pipeline for Free Spoken Digit dataset
+"""
+from typing import Any, Dict
+
+import luigi
+
+import hearpreprocess.pipeline as pipeline
+import hearpreprocess.tfds_pipeline as tfds_pipeline
+
+generic_task_config = {
+    "task_name": "spoken_digit",
+    "version": "1.0.9",
+    "embedding_type": "scene",
+    "prediction_type": "multiclass",
+    "split_mode": "new_split_kfold",
+    "nfolds": 5,
+    # mean: 0.41s, 75th: 0.47s, 90th: 0.56s, max: 2.28s
+    "sample_duration": 1.0,
```

Comment:
What is the 95th percentile? Maybe we shouldn't be extending this past 95th percentile and that is important?